protocol/x/perpetuals/types/keys.go

```diff
@@ -29,6 +31,9 @@ const (
 
 	// ParamsKey is the key to retrieve all params for the module.
 	ParamsKey = "Params"
+
+	// UpdatedOIKey is the key to retrieve the updated OI for the module.
+	UpdatedOIKey = "UpdatedOI"
```

Comment:
```suggestion
	UpdatedOIKeyPrefix = "UpdatedOI:"
```

Since this key is used as a prefix for each perp Id